usr/share/rear/layout/prepare/GNU/Linux/160_include_luks_code.sh

```diff
@@ -13,6 +13,9 @@ create_crypt() {
         value=${option#*=}
 
         case "$key" in
+            type)
+                cryptsetup_options+=" --type $value"
```

Comment:
Just off the top of my head:
I think the `--type` option is not supported by older cryptsetup versions
that is provided in older Linux operating systems
that are perhaps still supported by ReaR
so some backward compatibility code could be required here.
See in particular the sections
"Try hard to care about possible errors" and
"Maintain backward compatibility" and
"Dirty hacks welcome" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
I found it:
https://github.com/rear/rear/issues/2432#issuecomment-648099580

So for all supported SLES versions `cryptsetup` supports the `--type` option.

@vcrhonek 
did you check that for all supported RHEL and the other Red Hat based operating systems
`cryptsetup` supports the `--type` option?

Reply:
No, I will do.

Reply:
It's not supported on all systems, at least not at RHEL 6.

Reply:
@vcrhonek
if you do no longer need RHEL 6 support in current ReaR upstream master code
we can drop support for RHEL 6 and other Red Hat based operating systems
that are basically same for next ReaR version 2.7
cf. what I wrote about older operating systems in our release notes
https://github.com/rear/rear/blob/master/doc/rear-release-notes.txt#L2556
so we could drop support for RHEL 6 and related operating systems
with a note what is known to no longer work
i.e. LUKS support (also LUKS 1 for RHEL6) in this case.
When no LUKS is used `dmsetup ls --target crypt` should not output anything
so that usr/share/rear/layout/save/GNU/Linux/260_crypt_layout.sh
results nothing about LUKS in disklayout.conf.

Reply:
I've checked all Red Hat based operating systems and `type` option is supported on all of them except RHEL 6 and CentOS 6. I've also discovered that `blkid -p -o export` returns VERSION=256 for LUKS 1 on RHEL 6 so it won't get through LUKS version check anyway.